apiserver/common/networkingcommon/types.go

```diff
@@ -87,13 +87,28 @@ type BackingSpace interface {
 	// Name returns the space name.
 	Name() string
 
-	// Subnets returns the subnets in the space
-	Subnets() ([]BackingSubnet, error)
+	// NetworkSpace maps the space into network.SpaceInfo
+	NetworkSpace() (BackingSpaceInfo, error)
 
 	// ProviderId returns the network ID of the provider
 	ProviderId() network.Id
 }
 
+type BackingSpaceInfo struct {
```

Comment:
Don't need this; just have the backing return the `SpaceInfo`.

Reply:
Are you sure? The `Subnets` field needs to be of type []BackingSubnetInfo, not []SubnetInfo right?